QOLOE-22 Customising gutter value

ghazal-salehi opened the pull request.

stvp-qld reviewed (commented):
.

stvp-qld reviewed (changes requested):
Can we check/confirm the 40px footer requirement please. The left/right rail of footer should be consistent with header and body content.

ghazal-salehi commented:
The last change aligns the footer with qld-content. Example page: https://uat.forgov.qld.gov.au/user
Content page standard paint layout: https://oss-uat.clients.squiz.net/_admin/?screen=contents&assetid=471738

MitchAnderson93 reviewed (dismissed):
Looks good

duttonw commented:
Hi @ghazal-salehi ,

I've fixed the merge conflict and rebased your work to remove the noise. Can you please re-review your code changes and merge if its still correct.

Regards,

@duttonw

ghazal-salehi commented:
> Hi @ghazal-salehi ,
> 
> I've fixed the merge conflict and rebased your work to remove the noise. Can you please re-review your code changes and merge if its still correct.
> 
> Regards,
> 
> @duttonw

The files changed are as what expected. Merge was still blocked. I did re-request just now.

duttonw commented on src/components/bs5/footer/footer.scss:
```diff
@@ -5,14 +5,22 @@
 
+  > .container {
+    // overwriting bootstrap's default 1.5rem gutter value
+    --qld-gutter-x: 2.5rem;
```
this should be changed to (--#{$prefix} per the others

duttonw commented on src/components/common/layout/content.scss:
```diff
@@ -1,3 +1,8 @@
+
+    >.container {
+        // overwriting bootstrap's default 1.5rem gutter value
+        --qld-gutter-x: 2.5rem;
```
this should be changed to (--#{$prefix} instead of --qld 
so that prefix overrides are happy.

duttonw commented:
> > Hi @ghazal-salehi ,
> > I've fixed the merge conflict and rebased your work to remove the noise. Can you please re-review your code changes and merge if its still correct.
> > Regards,
> > @duttonw
> 
> The files changed are as what expected. Merge was still blocked. I did re-request just now.

This was due to a 'request changes' pr review, I've dismissed that review, but there is extra tweaks work that needs to be done to ensure this is not flakey.

ghazal-salehi commented:
The merge is blocked. Do you mind taking a look?

duttonw commented:
> The merge is blocked. Do you mind taking a look?

Hi @jortronQGov, @stvp-qld, @re-becca,

Please PR review this change and approve.

Regards,

@duttonw

duttonw reviewed (commented):
looks good, awaiting Online Experience team to provide approvals.

SenChung reviewed (approved):
Looks good.

elvishu reviewed (approved):
Looks good, Ghazal.
